packages/pages/src/common/src/project/structure.ts

```diff
@@ -71,25 +87,52 @@ export interface EnvVar {
  * @public
  */
 export interface ProjectStructureConfig {
-  filepathsConfig: ProjectFilepaths;
-  filenamesConfig: ProjectFilenames;
+  /** All important folders defined at the root of the project */
+  rootFolders: RootFolders;
+  /** All important folders within a specific root folder */
+  subfolders: Subfolders;
+  /** Files specific to the Yext configuration folder */
+  sitesConfigFiles: SitesConfigFiles;
+  /** Important files at the project's root */
+  rootFiles: RootFiles;
+  /** Defines how environment variables will be declared and processed */
   envVarConfig: EnvVar;
+  /**
+   * This is used for the case of multibrand setup within a single repo.
+   *
+   * The subfolder path inside src/templates and sites-config
+   * to scope a build to a subset of templates using a specific sites-config folder.
+   */
+  scope?: string;
 }
 
-export const defaultProjectStructureConfig: ProjectStructureConfig = {
-  filepathsConfig: {
-    templatesRoot: "src/templates",
-    functionsRoot: "src/functions",
-    sitesConfigRoot: "sites-config",
-    distRoot: "dist",
-    serverBundleOutputRoot: "assets/server",
-    renderBundleOutputRoot: "assets/render",
-    functionBundleOutputRoot: "functions",
+const DEFAULT_ASSETS_DIR = "assets";
+
+const defaultProjectStructureConfig: ProjectStructureConfig = {
+  rootFolders: {
+    source: "src",
+    dist: "dist",
+    sitesConfig: "sites-config",
+    functions: "functions",
+  },
+  subfolders: {
+    templates: "templates",
+    serverlessFunctions: "functions",
+    assets: DEFAULT_ASSETS_DIR,
+    serverBundle: "server",
+    renderBundle: "render",
+    renderer: "renderer",
+    static: "static",
+    plugin: "plugin",
+  },
+  sitesConfigFiles: {
+    ci: "ci.json",
+    features: "features.json",
+    siteStream: "site-stream.json",
+    serving: "serving.json",
   },
-  filenamesConfig: {
-    ciConfig: "ci.json",
-    featuresConfig: "features.json",
-    siteStreamConfig: "site-stream.json",
+  rootFiles: {
+    config: "config.yaml",
```

Comment:
not sure how the config.yaml stuff works yet but does this handle scoping?

Reply:
I think scope will be an attribute on the config, though not entirely sure. It shouldn't matter for this change though.